ballerina-shell/modules/shell-cli/src/test/java/io/ballerina/shell/cli/test/base/TestIntegrator.java

```diff
@@ -117,9 +116,7 @@ private String readResponse(BufferedReader stream) throws IOException {
      * Send the data given to the specific stream.
      */
     private void sendRequest(PrintStream stream, String string) throws InterruptedException {
-        stream.append(string);
-        stream.println(System.lineSeparator());
-        stream.flush();
+        stream.append(string).append("\n\n").flush();
```

Comment:
Why are replacing `System.lineSeparator` with `\n\n`?

Reply:
It also sent 2 newlines beforehand (one, since it is a `println` and one from the argument). This change actually fixed the testcases on windows, since jline would not detect `\r\n\r\n` as two newlines (I dont know why) but only `\n\n` on Windows and Linux. This code is only used inside testing, so I dont see any compatibility issues.

Reply:
Ack. Shall we include a comment explaining why we decided to keep "\n\n", noting that it is an issue on the JLine side. We can update this later once the issue is resolved.

Reply:
After revisiting this part, I noticed that it works on windows if there is only a single newline instead of two. There is probably a different handling of double newline at the end of the command between windows/linux by jline. I updated the code to just `stream.println(string);` which worked locally for me (on windows). Let's see if this also passes the Linux pipeline.

Reply:
Since the Ubuntu pipeline timed out twice, I reverted it to `\n\n` and added the comment.